src/gval/accessors/gval_xarray.py

```diff
@@ -173,18 +185,27 @@ def spatial_alignment(
 
         Parameters
         ----------
-        benchmark_map: Union[xr.Dataset, xr.DataArray]
+        benchmark_map: Union[gpd.GeoDataFrame, xr.Dataset, xr.DataArray]
             Benchmark map in xarray DataArray format.
         target_map: Optional[Union[xr.DataArray, xr.Dataset, str]], default = "benchmark"
             xarray object to match candidates and benchmarks to or str with 'candidate' or 'benchmark' as accepted values.
-        resampling : rasterio.enums.Resampling
+        resampling: rasterio.enums.Resampling
             See :func:`rasterio.warp.reproject` for more details.
+        rasterize_attributes: Optional[list], default = None
+            Attributes to rasterize from vector dataset
+
 
         Returns
         --------
         Union[xr.Dataset, xr.DataArray]
             Tuple with candidate and benchmark map respectively.
         """
+        benchmark_map = _rasterize_data(
```

Comment:
Shouldn't this be its own accessor method? I'm not sure if it's within the scope of spatial alignment as now spatial alignment has been given the task of data conversion as well? The other option is we rename this current accessor method to something like homogenize to better reflect the different operations that could take place?

Reply:
Changed to be dataframe accessor method.